WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -1013,6 +1016,27 @@ private extension ProductsViewController {
             .store(in: &subscriptions)
     }
 
+    func observePendingImageUploads() {
```

Comment:
```suggestion
    private func observePendingImageUploads() {
```

Reply:
This method is already inside a private extension so it's not necessary to mark it private again.